pkg/sqlstore/impl/system/sqlc.yaml

```diff
@@ -0,0 +1,29 @@
+version: "2"
+sql:
+  - schema: 
```

Comment:
This is a bit annoying. Could not find another solution. One of our queries touches the `sqlite_master` table and that's an internal table that has no schema definition inside `migrations` and `sqlc` needs a schema definition. 

We could add a `CREATE TABLE IF NOT EXISTS sqlite_master` inside `migrations`. That solves the `sqlc` problem but that fails when running the migration because `sqlite_master` is internal. 

So, the solution I found was to create another directory called `schemas` to [hold schemas](pkg/sqlstore/impl/system/schemas/schemas.sql) that will not be run by migration.

Reply:
Nice, good solution.